src/clarity/datagrid/_datagrid.clarity.scss

```diff
@@ -104,10 +105,21 @@
                 }
             }
 
+            .datagrid-column-flex {
+                display: flex;
+            }
+
+            clr-dg-filter,
+            clr-dg-string-filter {
+                display: flex;
+                order: 99;
+                margin-left: auto;
+            }
+
             &.asc, &.desc {
                 font-weight: 600;
 
-                &::after {
+                .datagrid-column-flex::after {
```

Comment:
Adding `display: flex` to this element is not needed, it doesn't need any display property set since the container being flew is all we need. But that's a nitpick, it doesn't change anything so this is good to go.